### PR TITLE
[NL NumberRange] Add basic .NET support for NumberRange model

### PR DESCRIPTION
**Pull request by iMicknl**
Still a work in progress, but already here for a long time. Would prefer to merge this already in, so others can build upon the current work.

Recognizer is disabled, only tests are running atm.

### CODE REVIEW

**Comment by tellarin** [comment]
It seems 25 tests are failing in .NET in the CI build. I'll start reviewing the current code soon.

File: .NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs [hunk 1]
```diff
@@ -274,13 +274,11 @@ protected override void InitializeConfiguration()
                     AgnosticNumberParserFactory.GetParser(AgnosticNumberParserType.Percentage, new DutchNumberParserConfiguration()),
                     new Dutch.PercentageExtractor(options)));
 
-            /*
             RegisterModel<NumberRangeModel>(
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs]: This is needed to run the corresponding tests, right? But we can't merge it in with the recognizer registered.

Comment (iMicknl) [reply .NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs]: I found this out also yesterday, after my PR... I will try to take some time for this this week, to clean it up and debug the failing tests. There are more tests failing than I thought.

Would prefer not to merge it in + enable tests, without almost full coverage.

File: Specs/Number/Dutch/NumberRangeModel.json [hunk 2]
```diff
@@ -286,24 +265,23 @@
   },
   {
     "Input": "Meer dan de helft van de mensen is aanwezig.",
-    "NotSupported": "dotNet",
     "Results": [
       {
-        "Text": "Meer dan de helft",
+        "Text": "meer dan de helft",
```
Comment (iMicknl) [inline Specs/Number/Dutch/NumberRangeModel.json]: @tellarin Should the NumberRange recognizer convert everything to lowercase? Or is this a bug and should the specs be changed back to first letter in uppercase.

File: Specs/Number/Dutch/NumberRangeModel.json [hunk 3]
```diff
@@ -286,24 +265,23 @@
...
+    "NotSupported": "dotNet",
     "NotSupportedByDesign": "javascript,python,java"
   },
   {
     "Input": "Meer dan 1/2 van de mensen is aanwezig.",
-    "NotSupported": "dotNet",
     "Results": [
       {
-        "Text": "Meer dan 1/2",
+        "Text": "meer dan 1/2",
```
Comment (iMicknl) [inline Specs/Number/Dutch/NumberRangeModel.json]: Same as `meer dan de helft`

**Comment by iMicknl** [comment]
@tellarin I did clean up the PR and all tests are ok. Disabled the recognizer (and the tests).

I will try to take some time for a follow up PR. I am not sure about some current implementations, regarding the boundaries. English also contains a different experimental NumberRange model. Dutch will need some work to support specific Dutch ways and to double check existing implementation.

For now;
Clippy_ShipIt

**Pull request merged**